elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -54,65 +63,68 @@ tar -xzf elasticsearch-oss-$ES_VERSION-linux-x86_64.tar.gz --strip-components=1
 cp -v opendistro-tar-install.sh $PACKAGE_NAME-$OD_VERSION
 
 # Install Plugin
+rm -rf /tmp/plugins
+mkdir -p /tmp/plugins
+
 for index in ${!PLUGINS_ARRAY[@]}
 do
-  if [ "$IS_CUT" = "true" ]
-  then
-    plugin_version=${CUT_VERSIONS_ARRAY[$index]}
-  fi
+  plugin_latest=`aws s3api list-objects --bucket $S3_RELEASE_BUCKET --prefix "${PLUGIN_PATH}${OD_VERSION}/$S3_RELEASE_BUILD/elasticsearch-plugins" --query 'Contents[].[Key]' --output text | grep -v sha512 | grep ${PLUGINS_ARRAY[$index]} | grep zip | sort | tail -n 1`
 
-  plugin_path=${PLUGINS_ARRAY[$index]}
-  plugin_latest=`aws s3api list-objects --bucket $S3_BUCKET --prefix "downloads/elasticsearch-plugins/${plugin_path}-${plugin_version}" --query 'Contents[].[Key]' --output text | sort | tail -n 1`
-
-  if [ "$plugin_path" != "none" ]
+  if [ ! -z "$plugin_latest" ]
   then
-    echo "installing $plugin_latest"
-    $PACKAGE_NAME-$OD_VERSION/bin/elasticsearch-plugin install --batch "${ARTIFACTS_URL}/${plugin_latest}"; \
+    echo "downloading $plugin_latest"
+    echo `echo $plugin_latest | awk -F '/' '{print $NF}'` >> /tmp/plugins/plugins.list
```

Comment:
What is the point of plugins.list for tarball?

Reply:
It was there in the previous code, I wanted to remove it, but was not sure if I was missing something. Will remove it.

Reply:
removed it.